### PR TITLE
show last updated only on static pages

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

"Last updated" should only appear on static pages. 

### How to review

Go to a static_landing_pages, timeseries and user_requested_data pages and see "Last updated". Pull this branch and "last updated should no longer appear on those pages but should only be on static pages. 

### Who can review

Anyone but me.

### CODE REVIEW

File: src/main/web/templates/handlebars/content/partials/intro-meta.handlebars [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{#if description.releaseDate}}
+{{#if_all type "static_page" description.releaseDate}}
```
Comment (dandwal) [inline src/main/web/templates/handlebars/content/partials/intro-meta.handlebars:1]: I am not sure this works, possibly looking for type_code "static_page"

**Review by dandwal — COMMENTED** [review]
I created a static landing page in Florence, published it then navigated to it and I still see the last updated banner.
(note the added text was me making sure I had the latest changes and it was compiling

Screenshot 2020-10-21 at 08 22 51

File: src/main/web/templates/handlebars/content/partials/intro-meta.handlebars (hunk 1 above)
Comment (NoelleDL) [reply src/main/web/templates/handlebars/content/partials/intro-meta.handlebars:1]: It's definitely not working on my machine the way I thought it was. Not sure what happened. Making the change.

**Review by dandwal — APPROVED** [review]
Have tested with both `static_landing_page`s and `static_page`s with the following results:
`static_page` show this banner
`static_landing_page` does not

As such I beleive this is working as expected along with clean code
Happy to approve.

**Review by dandwal — APPROVED** [review]
Reapproval

**Pull request merged**